libraries/tests/AWS.Lambda.Powertools.Idempotency.Tests/Handlers/IdempotencyFunction.cs

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Amazon.DynamoDBv2;
+using Amazon.Lambda.APIGatewayEvents;
+using Amazon.Lambda.Core;
+
+namespace AWS.Lambda.Powertools.Idempotency.Tests.Handlers;
+
+public class IdempotencyFunction
+{
+    public bool HandlerExecuted;
+
+    public IdempotencyFunction(AmazonDynamoDBClient client)
+    {
+        Idempotency.Configure(builder =>
+            builder
+                .WithOptions(optionsBuilder =>
+                    optionsBuilder
+                        //.WithUseLocalCache(true)
```

Comment:
if it's required uncomment it, otherwise remove it or move it to another unit test, if it's different usecase